main.go

```diff
@@ -85,6 +85,15 @@ func handler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+  // Stop users giving themselves karma.
+  if author == user {
+    amount = decreaseAmount(text)
+    s.Decrease(user, amount)
+    response.Text = author + " cheated themselves out of " + strconv.Itoa(amount) + " karma"
+    response.Send(*cliCallback)
+    return
+  }
+
```

Comment:
The code formatting police have cited you for not using consistent indentation in this file. The patch uses the proper 2-space-indentation. The rest of the file appears to use… my god, is that… no… *TABS*?

![huge-manatee](https://cloud.githubusercontent.com/assets/33429/8098296/ce4f7e88-1020-11e5-861c-7e2cb7721773.jpg)